setup.py

```diff
@@ -158,10 +161,8 @@ def build_extension(self, ext):
     },
     ext_modules=[
         CMakeExtension("torch_mlir._mlir_libs._jit_ir_importer"),
-    ],
-    install_requires=[
-        "numpy",
-        f"torch=={torch.__version__}".split("+", 1)[0],
-    ],
+    ] if not TORCH_MLIR_ENABLE_ONLY_MLIR_PYTHON_BINDINGS else [CMakeExtension("_")],
```

Comment:
you need something here for the built libs to actually be copied into the wheel; leaving `CMakeExtension("torch_mlir._mlir_libs._jit_ir_importer")` as is produces a spurious lib (basically nothing in it):

```shell
(venv) $ nm -gDC torch-mlir/_mlir_libs/_jit_ir_importer.cpython-311-x86_64-linux-gnu.so 
                 w __cxa_finalize
                 w __gmon_start__
                 w _ITM_deregisterTMCloneTable
                 w _ITM_registerTMCloneTable
```

I believe the alternative (correct-ish way) is to use a [MANIFEST.in](https://packaging.python.org/en/latest/guides/using-manifest-in/) but I haven't had much luck in the past getting that to work smoothly with MLIR Python bindings.